ssm/agents.py

```diff
@@ -379,3 +383,30 @@ def get_dns(dn_file, log):
 
     log.debug('%s DNs found.', len(dns))
     return dns
+
+
+def get_banned_dns(log):
+    """Retrieve the list of banned dns"""
+    banned_dns_list = []
+    try:
+        banned_dns_path = cp.get('auth', 'banned-dns')
```

Comment:
How is `cp` being accessed here? It looks like it's defined in a completely different scrop. Doesn't it need to be passed as an argument to this function?

Reply:
fixed